cmd/run.go

```diff
@@ -28,10 +28,14 @@ func dockerStop(imageName string) error {
 	return nil
 }
 
-func dockerRemove(imageName string) error {
+func containerRemove(imageName string, buildah bool) error {
```

Comment:
This should be `imageRemove()` shouldn't it?

Reply:
@ianpartridge - no, those are actually container removals. see:
https://github.com/appsody/appsody/blob/1475cd2acf8729273fd362b8e546a5b0c8104793/cmd/run.go#L31-L41

Reply:
Ah yes, sorry.  Thanks!